src/VisualStudio/Core/Def/Implementation/TableDataSource/MiscellaneousTodoListTable.cs

```diff
@@ -1,29 +1,24 @@
 ﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-using System.ComponentModel.Composition;
-using Microsoft.CodeAnalysis;
 using Microsoft.CodeAnalysis.Editor;
 using Microsoft.VisualStudio.LanguageServices.Implementation.ProjectSystem;
 using Microsoft.VisualStudio.Shell.TableManager;
 
 namespace Microsoft.VisualStudio.LanguageServices.Implementation.TableDataSource
 {
-    [Export(typeof(MiscellaneousTodoListTable))]
-    internal class MiscellaneousTodoListTable : VisualStudioBaseTodoListTable
+    internal sealed class MiscellaneousTodoListTable : VisualStudioBaseTodoListTable
```

Comment:
Same question as MiscellaneousDiagnosticListTable: what's special here to MiscellanousFilesWorkspace?

Reply:
(it looks the same as the VS one....)